src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int WORKING_DAY_DATE = 0;
+    private static final int WORKERS_NAME = 1;
+    private static final int WORK_TIME = 2;
+    private static final int PER_HOUR_SALARY = 3;
```

Comment:
```suggestion
    private static final int WORKING_DAY_DATE_INDEX = 0;
    private static final int WORKERS_NAME_INDEX = 1;
    private static final int WORK_TIME_INDEX = 2;
    private static final int PER_HOUR_SALARY_INDEX = 3;
```

by looking only on these constants it's unclear what is the purpose of them

Reply:
I agree